src/cmd/src/standalone.rs

```diff
@@ -532,6 +533,14 @@ impl StartCommand {
                 .context(OtherSnafu)?,
         );
 
+        // set the ref to query for the local flow state
+        {
+            let flow_worker_manager = flownode.flow_worker_manager();
+            information_extension
+                .set_flow_worker_manager(flow_worker_manager.clone())
+                .await;
```

Comment:
Same as above

Reply:
Why not move it up?  Before the creation of `catalog_manager`. And you don't need to clone the `information_extension` at all.

Reply:
> Why not move it up? Before the creation of `catalog_manager`. And you don't need to clone the `information_extension` at all.

because `flownode` need `catalog_manager` to translate sql and resolve table source, so there is a ugly cycle here(`a->b` means b need a)...:
```
information_extension -> catalog_manager -> flow_builder -> flownode -> flow_worker_manager -set-> information_extension
```
will later make flownode can set `catalog_manager` later in a separate PR so other struct's init can be easier